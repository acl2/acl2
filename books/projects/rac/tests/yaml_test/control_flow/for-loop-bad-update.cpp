// RAC begin

int loop() {

  int a = 0;
  for (int i = 0; i != 0; a += 1) {
  }
  return a;
}
