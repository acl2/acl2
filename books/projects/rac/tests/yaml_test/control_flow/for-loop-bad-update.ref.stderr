for-loop-bad-update.cpp:6 (26-32): The variable `i` used in the loop is never updated
 6 |   for (int i = 0; i != 0; a += 1) {
   |                           ^^^^^^
 7 |   }
